torch/_functorch/aot_autograd.py

```diff
@@ -911,6 +913,16 @@ def aot_module_simplified(
     # historically returned a function that was not the boxed calling
     # convention.  This should get fixed...
     def forward(*runtime_args):
+        if len(runtime_args) == 1 and all([isinstance(t, torch.Tensor) for t in runtime_args[0]]):
```

Comment:
Can we just... change the signature to `def forward(arg_list)`? I'm curious if that breaks anything downstream